src/Action/Store/AddAddress.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace AppBundle\Action\Store;
+
+use AppBundle\Entity\Store;
+
+
+class AddAddress
+{
+    public function __invoke(Store $data)
+    {
+        // TODO : we return the store instance but it maybe better to return the created address. to investigate : how to retreve the created address instance.
```

Comment:
No, it's fine to return the store, because the endpoint would return the store anyways.